static/Gruntfile.js

```diff
@@ -12,18 +12,17 @@ module.exports = function (grunt) {
     watch: {
```

Comment:
Github won't let me put a comment on the line I want, but we don't need to import webpackConfig or define a webpack task